Documentation/Installation/Environments/_dotenv-symfony.php

```diff
@@ -0,0 +1,12 @@
+<?php
+use Symfony\Component\Dotenv\Dotenv;
+
+try {
+    $dotenv = new Dotenv();
+    // Load file from ".env" file in the current directory (system/)
+    // You can of course specify another root directory here, like:
+    // $dotenv->load(TYPO3\CMS\Core\Core\Environment::getProjectPath() . '/.env');
+    $dotenv->load(__DIR__ . '/.env');
```

Comment:
That means that the .env file is under config/system folder? Mostly it should be in the project root folder.